core/embed/rust/src/ui/display/toif.rs

```diff
@@ -26,7 +26,11 @@ use super::Color;
 const TOIF_HEADER_LENGTH: usize = 12;
 
 pub fn icon(icon: &Icon, center: Point, fg_color: Color, bg_color: Color) {
-    let r = Rect::from_center_and_size(center, icon.toif.size());
+    icon_from_toif(&icon.toif, center, fg_color, bg_color);
+}
+
+pub fn icon_from_toif(toif: &Toif, center: Point, fg_color: Color, bg_color: Color) {
```

Comment:
What about naming the function `toif`? It doesn't really have anything to do with icons.

Reply:
Renamed in [16f01eb](https://github.com/trezor/trezor-firmware/pull/2610/commits/16f01eb1cfe9a9acf563e432474f966f3e53023a)